### PR TITLE
[TranslatorBundle] Added missing validation on translation file

### PR DESCRIPTION
**Pull request by krewetka**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | [2710](https://github.com/Kunstmaan/KunstmaanBundlesCMS/issues/2710)

### CODE REVIEW

**Comment by krewetka** [comment]
Hi everyone, 

I am not sure to which branch I should put this PR. Let me know then I can rebase it to correct one.

File: src/Kunstmaan/TranslatorBundle/Form/TranslationsFileUploadType.php [hunk 1]
```diff
@@ -19,6 +19,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder->add('file', FileType::class, [
             'required' => true,
             'label' => 'kuma_translator.form.upload_file_choose',
+            'constraints' => array(
```
Comment (acrobat) [inline src/Kunstmaan/TranslatorBundle/Form/TranslationsFileUploadType.php:22]: Can you use the short array syntax here? And add a trailing comma to the `new NotBlank()` line. Thanks!

File: src/Kunstmaan/TranslatorBundle/Form/TranslationsFileUploadType.php [hunk 2]
```diff
@@ -19,6 +19,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
...
+                new NotBlank()
```
Comment (acrobat) [inline src/Kunstmaan/TranslatorBundle/Form/TranslationsFileUploadType.php:23]: And there is a `use Symfony\Component\Validator\Constraints\NotBlank;` use missing

File: src/Kunstmaan/TranslatorBundle/Form/TranslationsFileUploadType.php (hunk 1 above)
Comment (krewetka) [reply src/Kunstmaan/TranslatorBundle/Form/TranslationsFileUploadType.php:22]: no problem :)  I see two types mixed in this file

**Comment by krewetka** [comment]
All should be fixed now. Only name in my branch in fork is 5.6 but it is rebased on 5.5

**Comment by krewetka** [comment]
@acrobat  are you able to figure what happened with travis build?

` Can't open ./.build/validate-composer.sh` does not look like related to my changes.

Also `The 'const_separation' fixer cannot be disabled because it is not enabled by your preset.` :thinking:

**Comment by acrobat** [comment]
@krewetka It seems that your 5.5 branch was not up to date with the latest changes as the `const_seperation fixer` error was fixed in https://github.com/Kunstmaan/KunstmaanBundlesCMS/commit/024385d34b3010257b29b96dbc521853190608e4 

And the `Can't open ./.build/validate-composer.sh` error does also seem to be related with the 5.6 branch as this check was only added since v5.6

**Comment by krewetka** [comment]
@acrobat  I rebased it on newest version of 5.5 and all checks are passed now. Only thing probably left is your approval.

**Review by acrobat — APPROVED** [review]

**Comment by acrobat** [comment]
Thanks @krewetka! And congrats on your first contribution! 🎉

**Pull request merged**